highfidelity/hifi
# Various Bot fixes

Fixes bugs related to recorded bots. Includes a better script for random bots in the theater.

Test Plan:
- Go to domain settings
- Using domain "New Assignment" ... Run https://rawgit.com/ZappoMan/hifi/89368afd68d3d87d9eafd7824462f05e56de7e19/script-archive/acScripts/BetterClientSimulationBotFromRecording.js
- Go to x: ~92-106, y: ~1, z: ~30-52
- notice there will be a bot animating
